docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/complexes.md

```diff
@@ -58,19 +58,15 @@ by Submodule with 1 generator
 1 -> x^4*e[1]
 
 julia> delta = map(C, 5)
-Map with following data
-Domain:
-=======
-Subquotient of Submodule with 1 generator
-1 -> e[1]
-by Submodule with 1 generator
-1 -> x^4*e[1]
-Codomain:
-=========
-Subquotient of Submodule with 1 generator
-1 -> e[1]
-by Submodule with 1 generator
-1 -> x^3*e[1]
+Module homomorphism
+  from subquotient of Submodule with 1 generator
+  1 -> e[1]
+  by Submodule with 1 generator
+  1 -> x^4*e[1]
+  to subquotient of Submodule with 1 generator
+  1 -> e[1]
```

Comment:
Maybe this should use `Indent()` ?
```suggestion
    1 -> e[1]
```

Reply:
Can't do that right now because it runs into a bug in pretty printing, see https://github.com/Nemocas/AbstractAlgebra.jl/issues/1561